tests/models/bert/test_modeling_flax_bert.py

```diff
@@ -161,3 +161,9 @@ def test_model_from_pretrained(self):
         model = FlaxBertModel.from_pretrained("bert-base-cased")
         outputs = model(np.ones((1, 1)))
         self.assertIsNotNone(outputs)
+
+
+class TestFlaxBertModelModuleIntegration(unittest.TestCase):
+    def test_clone(self):
```

Comment:
This test should work in the `FlaxBertModelTest` as a fast test no? Something like:
```python
def test_clone(self):
    config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
    for model_class in self.all_model_classes:
        model, params = model_class(config, _do_init=False)
        model = model.clone()
```

Reply:
I was just debugging using this test, I'll delete it.

Reply:
To be a little more clear, `apply` calls `clone` which internally creates a new instance according to dataclass semantics. Since I already fixed a test to use `apply`, this test is no longer needed.